tests/playwright/specs/wikipediaHomePage.js

```diff
@@ -1,14 +1,21 @@
+const { saveVideo } = require( 'playwright-video' );
+const path = require( 'path' ),
+	LOGPATH = process.env.LOG_DIR || path.resolve( __dirname, '..', 'log' );
+
 describe( 'Page', () => {
-	beforeAll( async () => {
+	it( 'should display sub title correctly', async () => {
+		let capture;
```

Comment:
capture is maybe too generic name. captureVideo might be better.

Reply:
This has been changed.